src/utils/MiscHelpers.js

```diff
@@ -165,6 +165,11 @@ export const getValueFromCss = (colorVar) => {
   return cssProps.getPropertyValue(`--${colorVar}`).trim();
 };
 
+/* Given a temperature in Celsius, returns value in Fahrenheit */
+export const celsiusToFahrenheit = (celsius) => {
+  return Math.round((celsius * 1.8) + 32);
```

Comment:
Why not :
```suggestion
  return Math.round((celsius * 9 / 5) + 32);
```
To keep style consistent with the below `fahrenheitToCelsius` ?

Reply:
I have no reason to prefer one form or the other.

Would you like me to make the suggested change, or are you asking merely out of curiosity?

If the latter, it is because I asked Google for the formula for the conversion, even though the reverse formula was staring me in the face and I might have worked it out myself.

If the former, I have no objections. Let me know if I should take action based on your suggestion.

Reply:
I was suggesting it for the style consistency. It's your PR, and your code is fine... My suggestion may be viewed as nitpicking ;-)